src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_NmTokens.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+using Xunit.Abstractions;
+using System.IO;
+using System.Xml.Schema;
+using System.Xml.XPath;
+
+namespace System.Xml.Tests
+{
+    #nullable enable
+    public class TC_SchemaSet_NmTokens : TC_SchemaSetBase
+    {
+        private void TestSchemaCompile(string fileName, bool negative)
+        {
+            string xsd = Path.Combine(TestData._Root, fileName);
+            XmlSchemaSet s = new XmlSchemaSet();
+            int numevents = 0;
+            s.ValidationEventHandler += (sender, args) => {
+                Assert.False(args.Severity == XmlSeverityType.Warning);
+                Assert.True(negative, args.Message);
+                numevents++;
+            };            
+            XmlReader r = XmlReader.Create(xsd);
+            s.Add(null, r);
+            s.Compile();            
+            Assert.False(negative && numevents != 1);
+        }
+
+        private void TestValidatedReader(string fileName, bool negative)
+        {
+            var settings = new XmlReaderSettings() {
+                ValidationType = ValidationType.Schema,
+                ValidationFlags = XmlSchemaValidationFlags.ProcessSchemaLocation | 
+                    XmlSchemaValidationFlags.ReportValidationWarnings,
+                XmlResolver = new XmlUrlResolver()
+            };
+            int numevents = 0;
+            settings.ValidationEventHandler += (sender, args) => {
+                Assert.True(negative, args.Message);
+                numevents++;
+            };
+            string xml = Path.Combine(TestData._Root, fileName);
+            using XmlReader r = XmlReader.Create(xml, settings);
+            XmlDocument doc = new XmlDocument();
+            doc.Load(r);
+            Assert.False(negative && numevents != 1);
+        }
+
+        [Fact]
+        public void Test_NmTokens_Schema()
+        {        
+            // Positive test: should not return any validation error during schema compile then default value is the list of NMTOKEN from enumeration
+            TestSchemaCompile("issue_60543_1.xsd", false);
```

Comment:
Nit: you could instead use the `TestSchemaCompile` and `TestValidatedReader`methods as [parameterized](https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/) tests

Reply:
Example:

```csharp
[Theory]
[InlineData("issue_xyz.xsd")]
[InlineData("issue_abc.xsd")]
public void Test_NmTokens_Schema(string fileName)
{
//...
}
```

Reply:
same for the test below